### ISSUE TITLE
[phet-io] ComboBox doesn't record state of selection menu or highlight

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#300]
@jonathanolson pointed out that in the state wrapper, comboBoxes are not saving the state of the open menu or highlight. As a result in the state wrapper, the downstream sim is only updated when an option is submitted. See below. @samreid have we discussed this before and decided these don't belong in the state of a sim?

![image](https://user-images.githubusercontent.com/6856943/27883371-c2a23c42-617c-11e7-8830-db141f4dece8.png)

**Comment by phet-steele** [comment]
@zepumph @jonathanolson @samreid see https://github.com/phetsims/phet-io/issues/1102#issuecomment-293361510 for why I haven't mentioned this before.

**Comment by zepumph** [comment]
Since opening this issue I talked with @samreid again about things like this. State is only for keeping track of the "meat and bones" of the sim. State's could be thought of as any configuration that a teacher may want to have on start up, so we don't are about partial states like an open combo box. Just like how we aren't keeping track of the phetMenu, since it is unlikely that teachers would want to open with the menu open or a dialog selected. The functionality is good as is. Thanks @phet-steele for doing some research for us. Closing.
